cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/internal/AliasedSSA.qll

```diff
@@ -131,6 +132,8 @@ abstract class MemoryLocation extends TMemoryLocation {
    * with automatic storage duration).
    */
   predicate isAlwaysAllocatedOnStack() { none() }
+
+  final predicate canReuseSSA() { any() }
```

Comment:
This would let a hypothetical fourth IR stage reuse anything from Aliased SSA IR, right? Maybe this should be `none()` until we need it to be otherwise.